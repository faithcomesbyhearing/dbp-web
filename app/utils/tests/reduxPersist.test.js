import reduxPersist from '../reduxPersist';

describe('redux persist utility config', () => {
	it('should match previous snapshot', () => {
		expect(reduxPersist).toMatchSnapshot();
	});
});
