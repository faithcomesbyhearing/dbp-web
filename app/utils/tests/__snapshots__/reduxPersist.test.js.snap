// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redux persist utility config should match previous snapshot 1`] = `
{
  "active": true,
  "reducerKey": "Bible.is",
  "reducerVersion": "1",
  "storeConfig": {
    "blacklist": [
      "homepage",
      "textSelection",
    ],
    "keyPrefix": "Bible.is.v1.",
    "stateReconciler": [Function],
    "whitelist": [
      "profile",
      "settings",
      "searchContainer",
      "notes",
    ],
  },
}
`;
